kong/resty/dns/client.lua

```diff
@@ -1167,9 +1161,6 @@ local function resolve(qname, r_opts, dnsCacheOnly, try_list)
     if try_list then
       -- check for recursion
       if try_list["(short)"..qname..":"..tostring(qtype)] then
```

Comment:
Should we add space between `..` operators?

Reply:
There is too much of this coding style here. I've only corrected the most obvious coding style issues that significantly affect the readability of the code. I'm not planning to fix the compact concatenation here for now. There is also a major issue with the naming style of variables and functions that I haven't fixed yet.